xpsi/utilities/ImportanceSample.py

```diff
@@ -26,31 +26,31 @@
 args = parser.parse_args()
 
 if not args.path_to_mains:
-    args.path_to_mains = input('Specify Absolute or relative path to main files: ')
+    args.path_to_mains = eval(input('Specify Absolute or relative path to main files: '))
 
 if not args.name_original_main:
-    args.name_original_main = input('Specify name of original main file: ')
+    args.name_original_main = eval(input('Specify name of original main file: '))
     
 if not args.name_importance_sample_main:
-    args.name_importance_sample_main = input('Specify name of importance sample main file: ')
+    args.name_importance_sample_main = eval(input('Specify name of importance sample main file: '))
 
 if not args.path_to_outputs:
-    args.path_to_outputs = input('Specify Absolute or relative path to output files: ')
+    args.path_to_outputs = eval(input('Specify Absolute or relative path to output files: '))
     if args.path_to_outputs[-1] =='/':
         args.path_to_outputs = args.path_to_outputs[:-1]
         
 if not args.output_root_name:
-    args.output_root_name = input('Specify name of output root file, without extension: ')
+    args.output_root_name = eval(input('Specify name of output root file, without extension: '))
 
 if not args.names:
-    names = input('Specify the name of the model variables: ') #check if input
+    names = eval(input('Specify the name of the model variables: ')) #check if input
     args.names = names.split()
 else:
     args.names = args.names[0].split()
 
 def check_prior_change(test):
     if not args.prior_change:
-        args.prior_change = input('Specify if importance sampling changes priors of model variables  (True/False): ')
+        args.prior_change = eval(input('Specify if importance sampling changes priors of model variables  (True/False): '))
```

Comment:
the `eval` can be left here (assuming the user does write "True" or "False"), but it also works without.